.eslintrc.json

```diff
@@ -33,6 +33,7 @@
     "react/sort-comp": 0,
     "react/prop-types": 0,
     "react/no-array-index-key": "off",
+    "react/destructuring-assignment": 0,
```

Comment:
I wouldn't switch off this rule. Anyway, I will check it manually and ask to use destructuring where you don't.